Gemfile

```diff
@@ -33,6 +33,8 @@ group :development, :test do
 end
 
 group :development do
+  gem 'better_errors'
```

Comment:
I checked in with @cnunciato about adding better_errors because I find it really helpful, and he said it was cool. I'm not aware of the Rails 4 functionality you describe.

[binding_of_caller](https://github.com/banister/binding_of_caller) is a gem often used with better_errors that allows you to look further down your call stack. Both are in account manage. 

If these need to be removed, I'm open to that but I find them really helpful.